controllers/imagerepository_controller.go

```diff
@@ -358,6 +359,22 @@ func (r *ImageRepositoryReconciler) ProvisionImageRepository(ctx context.Context
 			// Do not brake provision because of failed owner reference
 		}
 	}
+
+	if isComponentLinked(imageRepository) {
+		updateComponentAnnotation, updateComponentAnnotationExists := imageRepository.Annotations[updateComponentAnnotationName]
+		if updateComponentAnnotationExists && updateComponentAnnotation == "true" {
+			component.Spec.ContainerImage = quayImageURL
+
+			if err := r.Client.Update(ctx, component); err != nil {
+				log.Error(err, "failed to update Component after provision")
+				return err
+			} else {
+				log.Info("Updated component's ContainerImage")
```

Comment:
Maybe the log messages should at least mention the component name

Reply:
added